HW16/calculator.ts

```diff
@@ -0,0 +1,14 @@
+export function calculator(firstNumber: number, secondNumber: number, operation: '+' | "-" | '*' | '/' | '**', result: number) {
```

Comment:
Why do you need the forth argument `result` for?

Reply:
Corrected